packages/plugin-ext/src/plugin/status-bar-message-registry.ts

```diff
@@ -21,39 +21,82 @@ import {
 import { RPCProtocol } from '../api/rpc-protocol';
 import { StatusBarItemImpl } from './status-bar/status-bar-item';
 
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
 export class StatusBarMessageRegistryExt {
 
     proxy: StatusBarMessageRegistryMain;
 
+    protected readonly statusMessage: StatusBarMessage;
+
     constructor(rpc: RPCProtocol) {
         this.proxy = rpc.getProxy(Ext.STATUS_BAR_MESSAGE_REGISTRY_MAIN);
+        this.statusMessage = new StatusBarMessage(this);
     }
 
+    // copied from https://github.com/Microsoft/vscode/blob/6c8f02b41db9ae5c4d15df767d47755e5c73b9d5/src/vs/workbench/api/node/extHostStatusBar.ts#L122
```

Comment:
Is it typo?
L122 -> L174

Reply:
Is not obvious what was copied? I can add L174 if it is not.

Reply:
Ah, understood!

Reply:
fixed!